### ISSUE TITLE
Add shebang in scripts, write scripts as functions

### ISSUE THREAD

**Issue by chaomander2018** [UBC-MDS/DSCI_522_greenhouse_emissions_comparisons#27]
Hi @mkeyim @K3ra-y ,

I think we should follow Tiff's script organization. We might want to add shebang and write scripts as functions.

```#! /usr/bin/env Rscript 
# documentation comments

# import libraries/packages

# parse/define command line arguments here

# define main function
main <- function(){
    # code for "guts" of script goes here
}

# code for other functions goes here

# call main function
main()```

**Comment by chaomander2018** [comment]
Hi @K3ra-y @mkeyim, I edited the scripts following Tiffany's Rscript organization. As I know we don't need to write the run_all.sh,  I didn't add command line argument in them. What do you two think? Let me know, thank you!

**Comment by chaomander2018** [comment]
Hi, I have asked Tiffany and we will need write everything following her style. I will fixed the command line argument and push in a bit.

**Comment by chaomander2018** [comment]
Fixed by merge request #33
